### ISSUE TITLE
36k iso

### ISSUE THREAD

**Issue by yashar6909** [ch1x0r/LinuxRespin#12]
running the application with sudo, or directly with root using the command 
$respin dist 
or
$respin dist cdfs && respin dist iso custom.iso

both option in /home/respin/respin turn out with isos the size of 32 kb which isn't possible

**Comment by ferstar** [comment]
me too, 46Kb iso, sad!

**Comment by aeu79** [comment]
I had the same problem. There is no error in the command line, but you can find something in the log: genisoimage: Uh oh, I cant find the boot image 'isolinux/isolinux.bin' !
So, isolinux is missing...
Also got this error in the command line: /usr/bin/respin: línea 722: isohybrid: orden no encontrada
So installed syslinux-utils as well (not sure if it is completely needed).
Finally succeeded installing from scratch:
sudo add-apt-repository ppa:sergiomejia666/respin 
sudo add-apt-repository ppa:sergiomejia666/xresprobe 
sudo apt-get install xresprobe respin isolinux

And used it with:
sudo respin dist cdfs
(when is done, you can check/edit the files in the /home/respin/respin/isolinux folder to add a custom name, image, etc.) 
sudo respin dist iso custom.iso

Ubuntu 16.04 64 bits.
Source:
http://askubuntu.com/a/832793

**Comment by clessley** [comment]
Reference bug #16 ? Will test.

**Comment by ch1x0r** [comment]
So, in another thread, you will see the commenting out of the lightdm line. This seems to be a "new" issue as of Feb 8th or so. Working on getting a new release out to accommodate.

gksu "your text editor" respin

Comment out that lightdm line and should work.

**Comment by clessley** [comment]
Commenting out the line does not correct the issue. I'll keep my future comments in #16 since these are so tightly related. BTW, MATE uses lightdm as default.

**Comment by clessley** [comment]
Installed xresprobe deb, no issue, installed respin also through deb on Ubuntu Mate 16.04 = no lightdm error but 36k still iso.

I'm seeing on line 348 (almost at the bottom) of the respin.log the same as @aeu79 concerning `genisoimage: Uh oh, I can't find the boot image 'isolinux/isolinux.bin' !`

I will install as @aeu79 recommended in post three and try/test/report back.

**Comment by clessley** [comment]
I can confirm @aeu79 ppa's fix the problem. After adding the ppa's I reinstalled both xresprobe and the isolinux commands (since I used the .deb file before). I installed them each separately.

I command itself I used in respin was: `sudo respin backup gateway_ubuntu_mate_1604.iso`

I then used `sudo unetbootin` to create a live USB.

I was able to enter the live Ubuntu MATE 16.04 with my customized environment using Virtualbox on Windows 7. I did not have any login issue; this is an encrypted volume.

Oddly enough, the respun version doesn't include respin. I imagine this is to keep the iso file smaller and avoid duplicating iso files in the copy(?).

I am assuming at this point it would install on a real machine.
